Added information about environment variables used while running acceptance tests (#3821)
owncloud/docs: modules/developer_manual/pages/testing/acceptance-tests.adoc

When you are finished running the tests, remember to close the PR and leave a comment
describing the outcome of your testing.

=== Running Acceptance Tests Using Part System

> The part system allows us to divide the test run without knowing how many or which suites are available. Filter tags can also be used to run targeted parts after divisions.

Reviewer: ```suggestion
The part system allows us to divide the test run without knowing how many or which suites are available. Filter tags can also be used to run targeted scenarios after divisions. Multiple test suites are grouped together in each part. For example, if there are 26 test suites to be grouped into 10 parts then 2 or 3 test suites will be run in each part. This functionality is most useful in CI. The CI can split the test suites into as many pipelines as is appropriate without needing to know the actual names of the test suites.
```

The following two methods can be used to achieve this:

**1. Environment Variables**

[cols="20,80",options="header"]
|===
|Environment Variable
|Description

|`DIVIDE_INTO_NUM_PARTS`
|The number of parts the test suite will be divided into

|`RUN_PART`
|The part-number to test
|===

> Execute the tests by setting the above environments and running the usual make command.

Reviewer: ```suggestion
Execute the tests by setting the above environment variables and running the usual make command.
```

[source,bash]
----
RUN_PART=1 DIVIDE_INTO_NUM_PARTS=5 make test-acceptance-api
----

**2. part argument**

> If `run.sh` is used directly to run acceptance tests, part system can be achieved using `--part` flag. The script below divides the test suite into five parts and runs just the first one.

Reviewer: ```suggestion
If `run.sh` is used directly to run acceptance tests, the part system can be achieved using the `--part` flag. The script below divides the test suite into five parts and runs just the first one.
```
